syntax = "proto2";
package spider;

message consumer_register_req {
	required int32 id = 1;
}
